test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -713,7 +713,7 @@ public String getOptionsString() {
             }
 
             if (maxWriteSize != 0) {
```

Comment:
nit: Can we rename `maxWriteSize` and its methods in `AbstractIT` and in `CorfuReplicationLargeTxIT` to `maxReplicationWriteSize` to avoid confusion?